crc-admin-helper.spec.in

```diff
@@ -1,7 +1,7 @@
 # https://github.com/crc-org/admin-helper
 %global goipath         github.com/crc-org/admin-helper
 %global goname          crc-admin-helper
-Version:                0.5.4
+Version:                0.5.5
```

Comment:
wondering if this could be automated using [gitattributes](https://github.com/crc-org/vfkit/blob/main/.gitattributes) such as in vfkit: https://github.com/crc-org/vfkit/blob/main/pkg/cmdline/version.go#L12-L14
(but that's something to look at later :)